ui/package.json

```diff
@@ -33,7 +33,7 @@
     "@testing-library/react-hooks": "^5.1.0",
     "@testing-library/user-event": "^13.0.7",
     "css-mediaquery": "^0.1.2",
-    "prettier": "^2.2.1"
+    "prettier": "2.2.1"
```

Comment:
Why you changed the `prettier` dependency? Please revert `package.json` and `package-lock.json`

Reply:
Oh, I'm sorry. I must have accidently ran a npm command. Reverting.

Reply:
Ideally the fix would be to rebase interactively your branch (`git rebase -i`) and remove the unwanted changes, instead creating another commit reverting the files. But as I'm gonna squash the PR it does not really matter. Anyway, take a look at how rebase interactively works.

Reply:
I got it. I'll look into how rebase interactively. Thanks a lot!